modules/javafx.graphics/src/main/java/javafx/scene/layout/Background.java

```diff
@@ -353,6 +353,20 @@ public Background(final @NamedArg("fills") BackgroundFill[] fills, final @NamedA
         hash = result;
     }
 
+    /**
+     * A convenience factory method for creating a background with a single {@code Paint}.
```

Comment:
Minor: typo:  `background` -> `Background`
Let's use the `Background` in all places where we refer the class name. Two other places which need similar change are in @param and @return

A similar change is needed for Border.stroke method also: border -> Border

Reply:
Another suggestion for the description, inspired from the constructors of this class:
`A convenience factory method to create a new Background by supplying a single {@code Paint}.`

I am good with the current statement too. Above one will just be uniform with existing doc. I leave it to your choice.

Reply:
> Above one will just be uniform with existing doc.

In this case it doesn't matter much, but in general I don't like many places of the `Background` and `Border` docs as they stand :)
I already looked at them as a starting point when creating the initial commit.